assets/src/web-stories-block/block/index.js

```diff
@@ -22,44 +22,49 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { ReactComponent as icon } from './icon.svg';
-import metadata from './block.json';
 import deprecated from './deprecated';
-import edit from './edit';
 import save from './save';
+import { ReactComponent as icon } from './../images/icon.svg';
+import edit from './edit';
+import metadata from './block';
+import { BLOCK_TYPE_LATEST_STORIES, GRID_VIEW_TYPE } from './constants';
 import transforms from './transforms';
 
 const { name, category, attributes } = metadata;
 
 const settings = {
   title: __('Web Stories', 'web-stories'),
-  description: __('Embed a visual story.', 'web-stories'),
+  description: __('Embed Web Stories.', 'web-stories'),
   category,
   icon,
   keywords: [
```

Comment:
For some reason, I thought that keywords are limited at 3. But I am not sure where I got that from. Latest and selected feel like a little generic as keywords.